lua/entities/gmod_wire_expression2/core/custom/cfc_e2_lib.lua

```diff
@@ -0,0 +1,140 @@
+E2Lib.RegisterExtension("cfc_e2_lib", true)
+
+-- PvP Functions
+e2function number entity:playerIsInPvp()
+    if not IsValid(this) or not this:IsPlayer() then return 0 end
+    if this:GetNWBool("CFC_PvP_Mode", false) == true then return 1 end
+    return 0
+end
+
+e2function number entity:playerIsInBuild()
+    if not IsValid(this) or not this:IsPlayer() then return 0 end
+    if this:GetNWBool("CFC_PvP_Mode", false) == false then return 1 end
+    return 0
+end
+
+e2function string entity:getPvpStatus()
+    if not IsValid(this) or not this:IsPlayer() then return end
```

Comment:
This can't return `nil`, we have to default this to something. We could return `"unknown"` or `""`